db/schema.rb

```diff
@@ -250,6 +250,7 @@
     t.jsonb "meta_data", default: {}
     t.bigint "license_id"
     t.index ["license_id"], name: "index_tasks_on_license_id"
+    t.integer "access_level", limit: 2, default: 0, null: false, comment: "Used as enum in Rails"
```

Comment:
the migration add the this line before the `licence_id`-index for me
try to redo the migration or move this line up by yourself